### PR TITLE
meta-mender-nxp: Added support for imx8mq-voipac board

### PR DESCRIPTION
**Pull request by nandra**

### CODE REVIEW

**Comment by nandra** [comment]
> Just one nitpick. Any particular reason why this targets `dunfell`?

meta-voipac have only dunfell support

File: meta-mender-nxp/recipes-bsp/u-boot/u-boot-imx/0001-Added-mender-support.patch [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+diff --git a/include/configs/imx8mq_evk_voipac.h b/include/configs/imx8mq_evk_voipac.h
+index 9b9c266..532fb60 100644
+--- a/include/configs/imx8mq_evk_voipac.h
++++ b/include/configs/imx8mq_evk_voipac.h
+@@ -159,7 +159,7 @@
+ 	(CONFIG_SYS_INIT_RAM_ADDR + CONFIG_SYS_INIT_SP_OFFSET)
+ 
+ #define CONFIG_ENV_OVERWRITE
+-#define CONFIG_SYS_MMC_ENV_DEV		1   /* USDHC2 */
++#define CONFIG_SYS_MMC_ENV_DEV		0   /* USDHC2 */
```
[the remark this answers is not in the document]
Comment (nandra) [reply meta-mender-nxp/recipes-bsp/u-boot/u-boot-imx/0001-Added-mender-support.patch:41]: > That actually looks misleading. Shouldn't the comment be changed too?

You're right. I'll fix that.

File: meta-mender-nxp/recipes-bsp/u-boot/u-boot-imx/0001-Added-mender-support.patch [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+index 9b9c266..532fb60 100644
+--- a/include/configs/imx8mq_evk_voipac.h
++++ b/include/configs/imx8mq_evk_voipac.h
+@@ -159,7 +159,7 @@
+ 	(CONFIG_SYS_INIT_RAM_ADDR + CONFIG_SYS_INIT_SP_OFFSET)
+ 
+ #define CONFIG_ENV_OVERWRITE
+-#define CONFIG_SYS_MMC_ENV_DEV		1   /* USDHC2 */
++#define CONFIG_SYS_MMC_ENV_DEV		0   /* USDHC1 */
+ #define CONFIG_MMCROOT			"/dev/mmcblk1p2"  /* USDHC2 */
```
Comment (TheYoctoJester) [inline meta-mender-nxp/recipes-bsp/u-boot/u-boot-imx/0001-Added-mender-support.patch:42]: Sorry to be nagging, but is the `USBHC2` correct here then?

Comment (nandra) [reply meta-mender-nxp/recipes-bsp/u-boot/u-boot-imx/0001-Added-mender-support.patch:42]: > Sorry to be nagging, but is the `USBHC2` correct here then?

If you refer to COMFIG_MMCROOT variable comment I just keep it as it is as this variable is not used at all when mender is integrated.

**Review by TheYoctoJester — APPROVED** [review]

**Comment by TheYoctoJester** [comment]
@nandra merged, thanks. Just noticed that there is no entry in the README for the board. Do you want to add it, or should I?

**Comment by nandra** [comment]
Please add it. I've published already: https://hub.mender.io/t/voipac-imx8mq-industrial-development-kit/5865/1

**Pull request merged**